src/networking/bookings.ts

```diff
@@ -292,6 +292,11 @@ export const GET_GUEST_SORTED_BOOKINGS = gql`
   fragment baseTripFields on Booking {
     checkInDate
     checkOutDate
+    host {
+      id
+      firstName
+      supportEmail
```

Comment:
do we still need the support email if this feature lets the guest can contact the host via a form?

Reply:
ya i think we can deprecate `supportEmail`

Reply:
bump

Reply:
deleted all `supportEmail` declarations -- they were all in gql `networking` files